### PR TITLE
Leaks

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: core/src/main/java/org/fao/geonet/kernel/search/index/LuceneIndexLanguageTracker.java [hunk 1]
```diff
@@ -103,38 +101,30 @@ private void openIndex(String indexId) throws IOException {
...
-            IOUtils.closeQuietly(writer);
-            IOUtils.closeQuietly(cachedFSDir);
-            throw e;
-        } catch (IOException e) {
-            IOUtils.closeQuietly(nrtManager);
-            IOUtils.closeQuietly(writer);
-            IOUtils.closeQuietly(cachedFSDir);
-            throw e;
+            done = true;
+        } finally {
```
Comment (pmauduit) [inline core/src/main/java/org/fao/geonet/kernel/search/index/LuceneIndexLanguageTracker.java]: was there another exception that was not caught in the previous block ?

**Comment by pmauduit** [comment]
looks good

File: core/src/main/java/org/fao/geonet/kernel/search/index/LuceneIndexLanguageTracker.java (hunk 1 above)
Comment (pvalsecc) [inline core/src/main/java/org/fao/geonet/kernel/search/index/LuceneIndexLanguageTracker.java]: No idea and I don't think it matters.

Comment (pmauduit) [inline core/src/main/java/org/fao/geonet/kernel/search/index/LuceneIndexLanguageTracker.java]: I was just curious, nice factoring btw

**Pull request merged**